client/state/themes/selectors/get-theme-filter-string-from-term.js

```diff
@@ -11,12 +11,27 @@ import 'calypso/state/themes/init';
  * prefixed taxonomy:term
  * @param {Object} state Global state tree
  * @param {string} term The term slug
+ * @param {Array[string]} excludeTaxonomies List of taxonomies to exclude
+ * @param {Array[string]} includedTaxonomies List of taxonomies to include
  * @returns {string} Complete taxonomy:term filter, or empty string if term is not valid
  */
-export function getThemeFilterStringFromTerm( state, term ) {
+export function getThemeFilterStringFromTerm(
+	state,
+	term,
+	excludeTaxonomies = [],
+	includedTaxonomies = []
```

Comment:
Here and in `prependThemeFilterKeys`, could you call these two params with the same verb tense?
Either `exluded/included` or `exclude/include`.

Reply:
Done